### PR TITLE
Polymer 3 prep

### PR DESCRIPTION
**Pull request by margaree**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,25 +1,28 @@
 {
-  "name": "@d2l/accordion",
+  "name": "d2l-accordion",
   "version": "0.0.1",
```
Comment (dbatiste) [inline package.json]: Remove version?

File: package.json [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
   "repository": {
     "type": "git",
-    "url": "https://github.com/Brightspace/d2l-accordion"
+    "url": "https://github.com/BrightspaceUI/accordion"
   },
   "author": "D2L",
-  "license": "UNLICENSED",
+  "license": "Apache-2.0",
   "devDependencies": {
     "bower-art-resolver": "^2.0.6",
```
Comment (dbatiste) [inline package.json]: Is this needed?

File: wct.conf.json [hunk 3]
```diff
@@ -2,7 +2,49 @@
   "verbose": false,
   "plugins": {
     "local": {
-      "browsers": ["chrome"]
+      "browsers": ["chrome"],
+      "browserOptions": {
+        "chrome": ["headless", "disable-gpu", "no-sandbox"]
+      }
+    },
+    "sauce": {
```
Comment (dbatiste) [inline wct.conf.json]: Perhaps we can update so wct.conf.json is consistent with the rest...?

File: package.json (hunk 2 above)
Comment (jwong-d2l) [reply package.json]: used to be... probably not anymore

File: wct.conf.json (hunk 3 above)
Comment (jwong-d2l) [reply wct.conf.json]: is there a place to find the standard?  A template maybe?

Comment (dlockhart) [reply wct.conf.json]: https://github.com/Brightspace/generator-polymer-init-d2l-hybrid-element/tree/master/generators/app/templates

**Comment by margaree** [comment]
Does anyone have further feedback on this?

**Pull request merged**